_posts/2017-03-04-blog-on-gihub-with-jekyll-in-docker.markdown

```diff
@@ -1,42 +1,42 @@
 ---
 layout: post
-title:  "Blog on github pages with jekyll in docker for windows users"
+title:  "Blog on GitHub Pages with Jekyll in Docker for Windows users"
 date:   2017-03-04 16:00:00
 
-tags: 
+tags:
     - jekyll
     - github pages
     - docker
     - windows
 ---
 
-So you want to start blogging but do not know how. Let's discover on of the ways to do it using jekyll and github. Why github? Cause we're developers and it is the tool that we use everyday, so why not to use it for blogging as well?
+So you want to start blogging but do not know how. Let's discover one of the ways to do it using Jekyll and GitHub. Why GitHub? Because we're developers and it is the tool that we use everyday, so why not use it for blogging as well?
 
 
-## How to create a blog on github pages?
+## How to create a blog on GitHub Pages?
 
-Obviously you need to have a [github](https://gihub.com) account, if you still do not have one go get it.
+Obviously you need to have a [GitHub](https://github.com) account, if you still do not have one go get it.
 
-Once you have your account go to your profile and create new repository named <b>username.gihub.io</b>, where <b>username</b> is your github username.
+Once you have your account, go to your profile and create new repository named <b>username.gihub.io</b>, where <b>username</b> is your GitHub username.
 
-Now clone this repository to your machine and you're ready to create your site files and push it to gihub. Once it pushed it is available using http://username.gihub.io link
+Now, clone this repository to your machine and you're ready to create your site files and push them to GitHub. Once they are pushed, your site is available using http://username.gihub.io link.
 
 
 ## What is Jekyll?
 
-[Jekyll](http://jekyllrb.com) allows you to generate static website from your text files, usually markdown files. It is easy to use for blogging, it supports categories, posts, permalinks and has a lot of themes to apply for your blog. Moreover github pages are empowered by jekyll.
```

Comment:
I feel like *empowered* reads like *giving power to* and only applies to people where *powered* means *run with* or *run by*.